apps/interactor/tests/commander/commands/test_control.py

```diff
@@ -224,7 +224,13 @@ async def server(
         )
         for device in devices:
             if device is not kitchen_light:
-                devices.store(device).assertNoSetMessages()
+                assert devices.store(device).count(
```

Comment:
why?

Reply:
My best guess is to do with the caching of GetVersion and the fact that all the devices are Lights, so the only packet type that is resent regularly is GetColor and at this point in the testing, the values are stale.